### PR TITLE
Telescope map legend 

### PR DESCRIPTION
**Pull request by lydie10**
Hi @Theodlz @mcoughlin
This PR adds a legend to the telescope map. It's one of the features Sarah asked us to add and she confirmed that the labels were what she wanted. 
Thank you for your feedback and suggestions !

### CODE REVIEW

File: static/js/components/TelescopePageDesktop.jsx [hunk 1]
```diff
@@ -93,6 +131,32 @@ const TelescopePage = () => {
...
+        <div style={{ background: "#f9d71c" }} className={classes.circle} />
+        <p> Daytime</p>
+      </div>
+      <div className={classes.legend}>
+        <div style={{ background: "#0c1445" }} className={classes.circle} />
+        <p> Nighttime</p>
+      </div>
+      <div className={classes.legend}>
+        <div style={{ background: "#5ca9d6" }} className={classes.rect} />
+        <p> GCN and Networks </p>
```
Comment (mcoughlin) [inline static/js/components/TelescopePageDesktop.jsx:146]: Networks and Space-based Instruments

**Review by mcoughlin — APPROVED** [review]
Small wording suggestion.

File: static/js/components/TelescopePageDesktop.jsx (hunk 1 above)
Comment (lydie10) [reply static/js/components/TelescopePageDesktop.jsx]: okay, thanks I changed it

**Review by Theodlz — CHANGES_REQUESTED** [review]
Hi, @lydie10. I'm happy with the UI. However, the button to display the legend appears in a weird place for me.
image

(see, it's in the upper right corner.)

I would expect it to be closer to the map, let's say on the bottom right corner of it, rather than on the upper right of the form.

**Comment by lydie10** [comment]
Thanks to @ManonFlt the legend is now on the bottom right corner of the map

image

**Comment by Theodlz** [comment]
Hi @lydie10, there has been a breaking change in the python requirements that has been fixed a few days ago. Please pull new changes to your branch, so that the github actions can run and I can review your PR.

**Comment by Theodlz** [comment]
The code looks good. So once the github actions run and the tests pass, I'll give a green check and merge your PR.

**Pull request merged**